internal/cephfs/util.go

```diff
@@ -34,7 +34,7 @@ type volumeID string
 
 func execCommand(ctx context.Context, program string, args ...string) (stdout, stderr []byte, err error) {
 	var (
-		cmd           = exec.Command(program, args...) // nolint: gosec, #nosec
+		cmd           = exec.Command(program, args...) // #nosec:G204, commands executing not vulnerable.
```

Comment:
This would be vulnerable in case it is possible to call `execCommand()` with parameters that are not expected. Instead of only stating "not vulnerable", explain why that is the case (never called with user supplied parameters?)